### ISSUE TITLE
Fonction de génération d'un rapport des tableaux en latex

### ISSUE THREAD

**Issue by emilienschultz** [emilienschultz/pyshs-bib#17]
Objet/décorateur pour construire progressivement un rapport des traitements statistiques

**Comment by jeanbaptisteb** [comment]
si jamais cela peut aider: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_latex.html

**Comment by emilienschultz** [comment]
Je suis d'accord mais pour l'usage actuel des gens qui utilise pyshs, personne ne fait du latex, c'est excel ++ donc l'enjeu c'est la production d'un rapport intermédiaire sous un format Excel. J'imaginais plus quelque chose qui permettait d'identifier dans un notebooks les tableaux/commentaires à garder dans une sortie finale. Je ne suis pas encore sur que ça soit mûre comme idée :)
